### DISCUSSION TITLE
Code Coverage in the Cypress Dashboard

### DISCUSSION THREAD

**Discussion by joelgrimberg** [cypress-io/cypress#14562] [Cypress Cloud]
🎉 this is my first discussion on cypress 😃 

At the moment we are running Cypress with Code Coverage. works like a charm :-) 

somehow I would like to see my /coverage output in my Cypress Dashboard.
At the moment I have to go to CI (gitlab in my case) and browse the artifacts.

lets discusssss ;)

**Comment by bahmutov** [comment]
It would be kind of cool, wouldn't it

**Reply by joelgrimberg** [reply]
definitely ;-)

**Reply by joelgrimberg** [reply]
is this a first, or is this already part of the roadmap? ;-)

**Comment by s1mrankaur** [comment]
+1. Was surprised to not see it there. :(

**Comment by tamastompa** [comment]
+ 1
